<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COCSearcher</class>
 <widget class="QMainWindow" name="COCSearcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>COCSearcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image: url(&quot;app/ui/pic/background.jpg&quot;);
    background-position: center;
    background-repeat: no-repeat;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>906</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="maximumSize">
        <size>
         <width>720</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGraphicsView {
    background-image: url(&quot;app/ui/pic/logo.png&quot;);
    background-position: center;
    background-repeat: no-repeat;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mainlayout">
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    color: white;
    font-size: 40px;
    background-color: black;
    border: 2px solid white;
    border-radius: 10px;
	padding: 5px;
	margin-left: 20px;
}

QPushButton:focus {
    background-color: white;
    color: black;
}

QPushButton:!pressed {
    background-color: black;
    color: white;
}
</string>
           </property>
           <property name="text">
            <string>Старт</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>110</number>
           </property>
           <item>
            <widget class="QLineEdit" name="dark_elixir_input">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>700</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background: transparent;
    border: 2px solid white;
    border-radius: 20px;
    color: white;
    font-size: 20px;
    padding: 5px;
}

QLineEdit:focus {
    border: 2px solid black;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
    background-color: rgba(0, 0, 0, 175);
	border: 2px solid black;
	border-radius: 20px;
    color: white;
    font-size: 20px;
	margin-right: 120px;
	qproperty-alignment: AlignCenter;
}</string>
             </property>
             <property name="text">
              <string>Черный эликсир</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>110</number>
           </property>
           <item>
            <widget class="QLineEdit" name="elixir_input">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>700</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background: transparent;
    border: 2px solid white;
    border-radius: 20px;
    color: white;
    font-size: 20px;
    padding: 5px;
}

QLineEdit:focus {
    border: 2px solid #c001ff;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
    background-color: rgba(0, 0, 0, 175);
	border: 2px solid black;
	border-radius: 20px;
    color: white;
    font-size: 20px;
	margin-right: 120px;
	qproperty-alignment: AlignCenter;
}</string>
             </property>
             <property name="text">
              <string>Эликсир</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>110</number>
           </property>
           <item>
            <widget class="QLineEdit" name="gold_input">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>700</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background: transparent;
    border: 2px solid white;
    border-radius: 20px;
    color: white;
    font-size: 20px;
    padding: 5px;
}

QLineEdit:focus {
    border: 2px solid yellow;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
    background-color: rgba(0, 0, 0, 175);
	border: 2px solid black;
	border-radius: 20px;
    color: white;
    font-size: 20px;
	margin-right: 120px;
	qproperty-alignment: AlignCenter;
}</string>
             </property>
             <property name="text">
              <string>Золото</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
